### DISCUSSION TITLE
Ability to delete cloud backup

### DISCUSSION THREAD

**Discussion by prettycode** [iSoron/uhabits#1849] [Feature Requests] [answered]
On Android, if I delete all my habits, close the app, uninstall the app, and reinstall the app, instead of the app showing no habits when I open the app again, I see all the old habits that I deleted before uninstalling and reinstalling the app.

In my mind, it makes sense to have an item/action in the settings that allows for wiping the cloud backup.

**Comment by TheAmogh** [comment, marked as answer]
I'm guessing this is Android's Auto Backup that is backing up your app data. After you delete the habits, could you try running the Android backup manually so that the correct app data (without the habits) is synced with Google, and then reinstall the app.
